document/FileFormat.mdk

```diff
@@ -910,19 +938,25 @@ An `invocation` object contains information describing the invocation of the ana
 ### `commandLine` property { #invocation-commandLine }
 
 An `invocation` object may contain a property named `commandLine` whose value is a string
-containing the runtime parameters with which the tool was invoked.
-For command line tools, this string may consist of the completely specified command line used to invoke the tool.
+containing the completely specified command line used to invoke the tool,
+starting with the path to the tool's executable or script file.
```

Comment:
>starting with the path [](start = 0, length = 22)

The command-line doesn't need to start with the path, the user could simply invoke the tool or script name (which could result in the %PATH% var being consulted). Note that this is why we provide the filePath member, so that the log file can specify the complete path to the tool, if the path is resolved at runtime. The commandLine should precisely match the actual commandLine used to invoke the tool.